proto-public/pbconnectca/ca_grpc.pb.go

```diff
@@ -0,0 +1,178 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
+// versions:
+// - protoc-gen-go-grpc v1.2.0
+// - protoc             (unknown)
```

Comment:
I guess this is buf's doing.

Reply:
Yep, technically protoc isn't in use at all.